core/input/generisdata_emi.prn

```diff
@@ -39,15 +35,15 @@ pebiolc.seel.bioigccc         -20.0      20.0
 pebiolc.seel.bioigcc
 pebiolc.segabio.biogasc        -9.5       9.5
 pebiolc.seliqbio.bioftcrec    -12.0      12.0 
-pebiolc.seh2.bioh2c           -22.5      22.5   
-pebiolc.seel.biochp                             
-pebiolc.seh2.bioh2                              
-pebiolc.sehe.biohp                              
-pebiolc.sesobio.biotr                           
-pebiolc.sesobio.biotrmod                        
-pebiolc.segabio.biogas                          
-peoil.seliqfos.refliq          18.4               
-peoil.seel.dot                 18.4             0.01892
+pebiolc.seh2.bioh2c           -22.5      22.5   0.12614
```

Comment:
I think the N2O emissions need to be removed again - (see David's commits on this file to repair the wrong 3-fold tracking of n2o emissions)